pymagicc/io.py

```diff
@@ -1445,17 +1514,20 @@ def _convert_data_block_to_magicc_time(self, data_block):
         else:
 
             def _convert_to_decimal_year(itime):
-                # MAGICC dates are to nearest month at most precise
+                # MAGICC dates are either middle of start of month
                 year = itime.year
                 month = itime.month
-                _, month_days = monthrange(year, month)
-                day = itime.day
-                hr = itime.hour
-                month_fraction = (day + hr / 24) / month_days
-                # decide if midyear or start of year
-                if np.round(month_fraction, 1) == 0.5:
+
+                year_fraction = (itime - datetime(year, 1, 1)).total_seconds() / (
+                    datetime(year + 1, 1, 1) - datetime(year, 1, 1)
+                ).total_seconds()
+
+                startmonths = np.arange(0, 1, 1 / 12)
```

Comment:
So magicc uses a 360 day calendar?

Reply:
MAGICC doesn't have an internal concept of days so I wouldn't say it's a 360 day calendar. In #252 I phrase it as 'MAGICC has a year, divided into 12 equal length months'